src/nxdoc/nuxeo-add-ons/nuxeo-office-online-integration.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Nuxeo Office Online Integration
+description: 'The Nuxeo Office Online Integration addon adds the ability to view/edit Microsoft Office documents through Office Online.'
+review:
+    comment: ''
+    date: '2018-11-16'
+    status: ok
+labels:
+    - nuxeo-wopi
+toc: true
+tree_item_index: 1050
```

Comment:
Can you change this to 2400, so that it follows the alphabetical order in the left tree?